tests/admin_inlines/tests.py

```diff
@@ -1348,3 +1348,37 @@ def test_inline_formset_error(self):
         self.assertEqual(
             len(self.selenium.find_elements_by_css_selector(tabular_inline_formset_selector)), 1
         )
+
+    def test_inlines_verbose_name(self):
+        """
+        The item added by the "Add another XXX" link shows the correct verbose_name in the inline form.
+        """
+        self.admin_login(username='super', password='secret')
+        params = [
+            'admin:admin_inlines_course_add',
+            'admin:admin_inlines_courseproxy_add',
+            'admin:admin_inlines_courseproxy1_add',
+            'admin:admin_inlines_courseproxy2_add',
+        ]
+        css_selector = '.dynamic-class_set#class_set-%s h2'
+
+        for param in params:
+            with self.subTest(param=param):
+                self.selenium.get(self.live_server_url + reverse(param))
+
+                # First inline shows the verbose_name
+                available, chosen = self.selenium.find_elements_by_css_selector(css_selector % 0)
```

Comment:
Fomatting the `css_selector` in this way is a lot shorter, but I'm not so sure on readability. Part of me likes it as the difference is more clear as it avoids it being buried deep inside the string.